Civi/Api4/Generic/DAODeleteAction.php

```diff
@@ -83,6 +85,7 @@ protected function deleteObjects($items) {
         else {
           throw new \API_Exception("Could not delete {$this->getEntityName()} id {$item['id']}");
```

Comment:
You should use proper translate syntax if we are cleaning this up

Reply:
I thought we left error messages untranslated?

Reply:
Yeah maybe that's the case - Xavier did make a case for that at some point - @mlutfy ....

Reply:
Is the string supposed to be visible to users?

If not, OK to leave untranslated.